onnxruntime/test/contrib_ops/matmul_4bits_test.cc

```diff
@@ -141,8 +147,10 @@ TEST(MatMulNBits, Float32) {
     for (auto N : {1, 2, 32, 288}) {
       for (auto K : {16, 32, 64, 128, 256, 1024, 93, 1234}) {
         for (auto block_size : {16, 32, 64, 128}) {
-          RunTest(M, N, K, block_size, false, false);
-          RunTest(M, N, K, block_size, true, false);
+          for (auto comp : {CompFp32, CompInt8}) {
```

Comment:
Could you add add `CompUndef` for `comp` as well?

Reply:
added